assembly/resources/application.yml

```
@@ -1,33 +1,84 @@
-#camunda.client:
-# mode: self-managed
-# grpc-address: http://localhost:26500
-# rest-address: http://localhost:8088
-
-migrator:
-  rdbms-exporter:
-    auto-ddl: true
-  c7:
-    auto-ddl: true
-  source:
-    jdbc-url: jdbc:h2:./h2/data-migrator-source.db;TRACE_LEVEL_FILE=0;DB_CLOSE_ON_EXIT=FALSE
-    username: sa
-    password:
-    driver-class-name: org.h2.Driver
-  target:
-    jdbc-url: jdbc:h2:./h2/data-migrator-target.db;TRACE_LEVEL_FILE=0;DB_CLOSE_ON_EXIT=FALSE
-    username: sa
-    password:
-    driver-class-name: org.h2.Driver
-    
-  # batch-size: 500
+camunda:
+  # Client configuration for connecting to Camunda 8 for runtime migration
+  client:
+    # Operation mode of the client ('self-managed' or 'cloud')
+    # mode: self-managed
+
+    # The gRPC API endpoint for Camunda 8 Platform
+    # grpc-address: http://localhost:26500
+
+    # The REST API endpoint for Camunda 8 Platform
+    # rest-address: http://localhost:8088
+
+  migrator:
+    # Number of records to process in each migration batch
+    # batch-size: 500
+
+    # Automatically create/update database schema
+    # auto-ddl: true
+
+    # Optional prefix for migrator database schema tables
+    # table-prefix:
```

```suggestion
    # Automatically create/update database schema
    auto-ddl: true

    # Optional prefix for migrator database schema tables
    # table-prefix: MY_PPREFIX_
```

/suggest We should configure `auto-ddl` for the migrator to `true` since in most cases the table doesn't exist.